Aptoide/appcoins-wallet-android
# [MOB-249] rakam wallet events

**What does this PR do?**

This PR implements rakam in wallet and the new analytics being sent with it.

**Database changed?**

No

**Where should the reviewer start?**

- [ ] App.java
- [ ] PaymentMethodsFragment.java

**How should this be manually tested?**

Check the tickets/confluence page and make sure all the events are being sent correctly;

Confluence: https://aptoide.atlassian.net/wiki/spaces/MOB/pages/1586233345/Rakam+-+Metrics

**What are the relevant tickets?**

  Tickets related to this pull-request: [MOB-249](https://aptoide.atlassian.net/browse/MOB-249)

**Code Review Checklist**
